mlflow/llama_index/tracer.py

```diff
@@ -293,3 +340,99 @@ def _extract_token_usage(
                     if (v := additional_kwargs.get(k)) is not None:
                         usage[k] = v
             return usage
+
+
+_StreamEndEvents = Union[LLMChatEndEvent, LLMCompletionEndEvent, ExceptionEvent]
```

Comment:
```suggestion
_StreamEndEvent = Union[LLMChatEndEvent, LLMCompletionEndEvent, ExceptionEvent]
```

`events` sounds like this type represents a collection of events.

Reply:
Makes sense!